src/databricks/labs/ucx/install.py

```diff
@@ -652,16 +652,92 @@ def install_on_account(self):
         # upload the json dump of workspace info in the .ucx folder
         ctx.account_workspaces.sync_workspace_info(installed_workspaces)
 
+    def join_collection(
+        self,
+        current_workspace_id: int,
+    ):
+        if not self.is_account_install and self.prompts.confirm(
+            "Do you want to join the current installation to an existing collection?"
+        ):
+
+            installed_workspaces: list[Workspace] | None = []
+            account_client = self._get_safe_account_client()
+            ctx = AccountContext(account_client)
+            try:
+                accessible_workspaces = ctx.account_workspaces.get_accessible_workspaces()
+                collection_workspace = self._get_collection_workspace(accessible_workspaces, account_client)
+                if collection_workspace is not None:
+                    installed_workspaces = self._sync_collection(
+                        collection_workspace, current_workspace_id, account_client
+                    )
+                if installed_workspaces is not None:
+                    ctx.account_workspaces.sync_workspace_info(installed_workspaces)
+            except PermissionDenied:
+                logger.error("User doesnt have account admin permission, cant join a collection, skipping...")
```

Comment:
this would fail the installation, need `logger.warning`
```suggestion
                logger.warning("User does not have account admin permission, cannot join a collection, skipping...")
```